modules/tutorials/pages/tutorial-install-server.adoc

```diff
@@ -0,0 +1,63 @@
+= Install the Couchbase Server
+:description: Install and set up the Couchbase Server to continue following the Student Record System tutorial.
+:page-topic-type: tutorial
+:imagesdir: ../images
+:page-pagination: full
+:page-toclevels: 2
+:page-aliases: install-couchbase-server.adoc
+
+:sdk-footnote: footnote:[Software Development Kit]
+
+[abstract]
+{description}
+
+== Installation
+
+You can install and run the Couchbase Server as a standalone application or as a Docker instance.
+
+=== Standalone
+
+To download the Couchbase Server as a standalone application:
+
+. Go to the https://www.couchbase.com/downloads/[Couchbase download page].
+. Go to menu:Server[Couchbase Server Community].
+. Select a **Version** and an **OS** and click btn:[Download].
+
+After the Couchbase Server is finished downloading, follow the instructions to install it on your machine.
+
+=== Docker
+
+To download the Couchbase Server as a Docker instance:
+
+. Download https://www.docker.com/get-started/[Docker^].
+. Open a terminal window and run this command to provision the server image, set up the storage location, and run the server:
++
+[source, sh]
+----
+docker run -t --name db -p 8091-8096:8091-8096 -p 11210-11211:11210-11211 \
+couchbase/server:enterprise-{current-enterprise-version}
+----
+
+
+== Set Up a Cluster
+
+A Couchbase Server can run as its own cluster or join another cluster to form a multi-node system.
+For this tutorial, your server runs as a single node.
+
+To set up a cluster:
+
+. Go to your server's configuration page.
+You can access this page in any of the following ways:
+* Opening your local Couchbase Server app.
+* Running a Docker instance in the terminal.
+* Going directly to `pass:[http://127.0.0.1:8091]` in your browser.
+. Click btn:[Setup New Cluster].
+. For your cluster name, enter **student-cluster**.
+. Leave the username as **Administrator** and set the password as **password**.
```

Comment:
I wouldn't recommend telling people a specific username and password - this could expose security issues. 

Tell them to set a username and password, and in any places you've used those values in the code samples, just replace with placeholder values.